Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -156,6 +167,10 @@ public void RegisterPointer(IPointingSource pointingSource)
                 {
                     gazeManagerPointingData = new PointerData(pointingSource);
                 }
+                else
+                {
+                    gazeManagerPointingData.ResetFocusedObjects();
```

Comment:
I was just about to add a message about that :).  I think that should be the default behavior.  Are you OK with me making the main InputManagerTest scene 6DoF-compatible and make the main InputManager prefab include the SimpleSinglePointerSelector and be configured correctly for 6DoF or GGV?